pkg/api/authz.go

```diff
@@ -194,7 +194,18 @@ func AuthzHandler(ctlr *Controller) mux.MiddlewareFunc {
 
 			/* To do: verify client certs and get its username(subject DN)
 			if request.TLS.VerifiedChains != nil, then get subject DN
-			issue: https: //github.com/project-zot/zot/issues/614 */
+			issue: https://github.com/project-zot/zot/issues/614 */
+
+			if request.TLS.VerifiedChains != nil {
+				
+				for _, cert:=range request.TLS.PeerCertificates {
+					acCtrlr.Log.Info().Msgf("Certificate elements: %#v", cert.Subject.CommonName)
+				}
```

Comment:
here we need to take the username from request.TLS.PeerCertificates or request.TLS.VerifiedChains